extensions/adobe/experience/adcloud/searchads/transaction.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/adcloud/searchads/transaction",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Search Advertising Transaction Property",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "meta:auditable": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Search Advertising Transaction Class includes fields that defines Transaction.",
+  "definitions": {
+    "searchads-transaction": {
+      "properties": {
+        "searchads:recordId": {
```

Comment:
It looks like you're inheriting the "record" class for this schema. 

https://github.com/adobe/xdm/blob/deb68f399a1682b780f54edf421013b6b95c3c65/extensions/adobe/experience/adcloud/searchads/adgroup.schema.json#L128

Records have an @id field that we can use for a unique field that otherwise lacks business-context. (though uniqueness is not enforced on the @id field). @jishah-adobe , would it be better to use the id field from records?

Reply:
Rename to uniqueTransactionId